apps/cowswap-frontend/src/modules/bridge/pure/contents/QuoteSwapContent/index.tsx

```
@@ -0,0 +1,92 @@
+          Recipient <InfoTooltip content="The address that will receive the tokens." size={14} />
+        </>
+      ),
+      content: <RecipientDisplay recipient={recipient} chainId={sellAmount.currency.chainId} />,
+    },
+  ]
+
+  return (
+    <>
+      <TradeFeesAndCosts receiveAmountInfo={receiveAmountInfo} />
+      {contents.map(({ withTimelineDot, label, content }) => (
+        <ConfirmDetailsItem withTimelineDot={withTimelineDot} label={label}>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)